samples/Extensions/EventHubs/EventHubsTriggerMetadata.cs

```diff
@@ -50,5 +52,44 @@ public static void UsingContext([EventHubTrigger("src-context", Connection = "Ev
                 Dictionary<string, JsonElement> systemProperties = systemPropertiesArray[i];
             }
         }
+
+        // batched scenarios
+        [Function("EventHubsTriggerMetadata-ByteParameter-Batched")]
+        public static void UsingByteParameterBatched([EventHubTrigger("src-parameters", Connection = "EventHubConnectionAppSetting")] IEnumerable<byte[]> batchedMessages,
```

Comment:
We might have a 🐛 in the binding pipeline which causes batchedMessages to be null.

Reply:
Is the issue solved? there's a workaround?